great_expectations/render/util.py

```diff
@@ -241,6 +241,37 @@ def handle_strict_min_max(params: dict) -> tuple[str, str]:
     return at_least_str, at_most_str
 
 
+def _get_value_to_render(value_: Any) -> Any:
+    if value_ is not None and value_ != "":
+        return value_
+    if value_ == "":
+        return "EMPTY"
+    return "null"
+
+
+def _get_header_row(all_unexpected_in_samples: bool) -> list[str]:
+    # Check to see if we have *all* of the unexpected values accounted for. If so,
+    # we show counts. If not then we show Sampled Unexpected Values only.
+    if all_unexpected_in_samples:
+        header_row = ["Unexpected Value", "Count"]
+    else:
+        header_row = ["Sampled Unexpected Values"]
```

Comment:
We also remove "Count" from the header (otherwise an empty column would render).